src/EventStore.ClusterNode/styles/app.css

```
@@ -1,11 +1,11 @@
 @import "tailwindcss/theme.css" layer(theme);
 @import "tailwindcss/utilities.css" layer(utilities) source(none);
 
-@source "../clusternode-web/js/admin-operations.js";
-@source "../clusternode-web/js/cluster-status.js";
-@source "../clusternode-web/js/queue-dashboard.js";
-@source "../clusternode-web/js/stream-browser.js";
-@source "../clusternode-web/js/ui-auth.js";
+@source "../ui-assets/js/admin-operations.js";
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)